2020-05-22 IAC Team Blog Post (#64)
puppetlabs/iac: _posts/2020-05-22-status-update.md

---
layout: post
title: "22nd May 2020: IAC Team Status Update"
author: pmcmaw
categories:
  - team
  - status
tags:
  - module updates
  - community
  - testing
  - haproxy
  - motd
  - vmpooler
  - abs
  - pdksync
  - pdk
  - honeycomb
  - kubernetes
  - mysql
  - docker
  - pdksync
  - pdk

---
The IAC team has been working on a lot of new things over the past week. We want to share with our community some pretty interesting things. Here's a brief overview of what we've been up to:

> ## Diana has been made permanent!
> We are absolutely delighted to announce our current intern Diana has recently been offered a permanent job role. Diana has been a major influence on our community reporting tool for when we are triaging PRs. We are so excited that she will be hanging around! We are all looking forward to have a celebratory beverage together at some point in the future.

Reviewer: ```suggestion
## Daiana has been made permanent!
We are absolutely delighted to announce our current intern, Daiana, has recently been offered a permanent job role. Daiana has been a major influence on our community reporting tool for when we are triaging PRs and releasing modules. We are so excited that she will be hanging around! We are all looking forward to have a celebratory beverage together at some point in the future.
```

## Community Contributions

We think that is no doubt about our community contributors, they are always ready to help and they have excellent ideas. Here are some highlights of this week:

Thank you to Trey Dockendorf for raising an issue on the latest release of the PDK, and to [Jesse Scott](https://github.com/scotje) for providing a [fix](https://github.com/puppetlabs/pdk/pull/883). [David](https://github.com/DavidS) has now merged this and this fix will make it out in the next release.

Thank you to [Victor Cabezas](https://github.com/Wiston999) for a fantastic PR for [puppetlabs/kubernetes](https://github.com/puppetlabs/puppetlabs-kubernetes). Its great to see such attention to detail in a PR from a community member.

Thank you to [andeman](https://github.com/andeman)for contributing to our [puppetlabs/mysql](https://github.com/puppetlabs/puppetlabs-mysql) module and providing us with such a great feature.

Thank you to [https://github.com/vchepkov](Vadym Chepkov) for your [Docker PR](https://github.com/puppetlabs/puppetlabs-docker/pull/613). Housekeeping on our modules is always beneficial and we really appreciate you taking the time to contribute.

## PDKSync Runs

>  [David](https://github.com/david22swan) has ran 2 PDKSync runs this week to include generic template updates in our modules and to add Ubuntu 20.04 support and [Danny](https://github.com/carabasdaniel) has been running one to switch our default provisioner from vmpooler to abs.

Reviewer 2: ```suggestion
[David](https://github.com/david22swan) has performed 2 PDKSync runs this week to include generic template updates in our modules and to add Ubuntu 20.04 support. [Danny](https://github.com/carabasdaniel) has been running one to switch our default provisioner from vmpooler to abs.
```

>  [David](https://github.com/david22swan) has ran 2 PDKSync runs this week to include generic template updates in our modules and to add Ubuntu 20.04 support and [Danny](https://github.com/carabasdaniel) has been running one to switch our default provisioner from vmpooler to abs.

Reviewer: ```suggestion
 [David](https://github.com/david22swan) has ran 2 PDKSync runs this week to include generic template updates in our modules and to add Ubuntu 20.04 support and [Danny](https://github.com/carabasdaniel) has been running one to switch our default provisioner from VMPooler to ABS for our weekly and pre-release CI runs. 
```

[Ciaran](https://github.com/sanfrancrisko) now knows his neighborhood by the back of his hand, has taken up more cooking opportunities at home (he currently has enough Bolognese in his freezer to feed the whole IAC team), is getting a big nostalgia hit playing Streets of Rage 4 online with his mate and has big plans on getting back into playing guitar, it just hasn’t happened yet. Sounds like Ciaran has been doing great at keeping busy!

[Danny](https://github.com/carabasdaniel) has found himself spending more time chilling out and reading books and when he is not doing that he is ‘fine tuning’ his video gaming skills.

[David](https://github.com/david22swan) carries out a vigorous home workout regime on Tuesday, Wednesday and Thursdays. Way to go David, serious commitment there!

[Sheena](https://github.com/sheenaajay) is currently home schooling 2 lovely girls. Although when they have some down time she is exploring new recipes with them.

> [Diana](https://github.com/daianamezdrea) has started to do some home workouts and has started eating healthy I wonder is this a result of mastering how to bake a perfect lava cake? I also wonder how shipping a lava cake would work!?

Reviewer 2: ```suggestion
[Diana](https://github.com/daianamezdrea) has started to do some home workouts and has started eating healthy. I wonder is this a result of mastering how to bake a perfect lava cake? I also wonder how shipping a lava cake would work!?
```

> [Diana](https://github.com/daianamezdrea) has started to do some home workouts and has started eating healthy I wonder is this a result of mastering how to bake a perfect lava cake? I also wonder how shipping a lava cake would work!?

Reviewer: ```suggestion
[Diana](https://github.com/daianamezdrea) has started to do some home workouts and has started eating healthy. I wonder if this is a result of mastering how to bake a perfect lava cake? I also wonder how shipping a lava cake would work!?
```

Reviewer 2: ...and one more `s/Diana/Daiana/` there too 😉
